### DISCUSSION TITLE
A lot of distortion on the output

### DISCUSSION THREAD

**Discussion by flippthebunny** [cardonabits/haxo-hw#43] [Q&A] [answered]
Hello,

I got the haxophone today, am using a pi zero wh, and it does turn on and seem to work sorta however the audio output is extremely noisy. There's a ton of hiss coming out of the speaker. Also, the instrument when it does generate audio sounds extremely distorted. I have no idea what I could be doing wrong?

**Comment by jcard0na** [comment, marked as answer]
Hello,

I am really sorry that you are having issues with the audio quality on the
Haxophone.
What type of headphone or speaker are you using?  If using very low
impedance speakers or headphones they need to be capable of handling
sufficient power.  Please see this section in our last update
<https://www.crowdsupply.com/cardona-bits/haxophone/updates/haxophones-on-the-move#headphone-considerations>
and let me know if that helps.
Are you able to test with a different speaker?

Best,
Javier Cardona

**Comment by jcard0na** [comment]
Hi @flippthebunny,

Were you able to fix the distortion with a different headphone/speaker?

If you always expect to use your haxophone with very low power headphones, you can reduce the amp gain by half by soldering a bridge between positions 2 and 3 on jumper JP1 as shown below.  

![IMG_3337](https://github.com/cardonabits/haxo-hw/assets/676181/ca72de01-5826-4515-b5de-1ecc9558f240)

This will lower the output power by half (3 dB).

Please let me know how that goes.
Thanks!

Javier

**Comment by flippthebunny** [comment]
Hey there!

I am sure it must have been the speaker I was using. I may actually do that
mod cuz I was thinking a portable speaker would be a good idea. I might try
a few others I have first lol

I honestly haven't had time to mess with it since. Holidays and a small
emergency took over. I will this week :) I'm excited to get it going
properly!

Thanks for the response, I will let you know how it goes.

**Comment by jcard0na** [comment]
Excellent, thanks for the update.  Take care of emergencies first... haxophone second :wink:
